src/Hosting/Hosting/src/Internal/HostingMetrics.cs

```diff
@@ -22,24 +22,27 @@ public HostingMetrics(IMeterFactory meterFactory)
 
         _currentRequestsCounter = _meter.CreateUpDownCounter<long>(
             "current-requests",
-            description: "Current requests");
+            description: "Number of concurrent HTTP requests that are currently active on the server.");
 
         _requestDuration = _meter.CreateHistogram<double>(
             "request-duration",
-            description: "Request duration");
+            unit: "s",
+            description: "The duration of HTTP requests on the server. Request duration is measured in seconds.");
```

Comment:
Remove the unit part from the description since a) unit is already provided as another explicit argument b) A common practice in OpenMetrics/Prometheus/OpenTelemetry - metrics description is more focusing on the intention:

```suggestion
            description: "The duration of HTTP requests on the server.");
```

Ref: https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#overall-structure

![image](https://user-images.githubusercontent.com/17327289/230094935-a4176544-90dc-4669-bde9-e5b26cd1546e.png)

Reply:
In addition (I know it's a hard problem), it will be great to clarify what does "duration of HTTP requests on the server" mean - e.g. is this the duration measured on L7, HTTP.SYS (or the Linux equivalent), IIS pipelines, ASP.NET Core layer or something else.

And more importantly, once the duration is established, make sure that in the future releases / architecture changes, the way how things are measured is comparable with previous versions; Or alternatively, set a clear & low expectation.

Here is a great example of "what exactly are we talking about when it comes to navigation latency/timing" https://www.w3.org/TR/navigation-timing/.

![image](https://user-images.githubusercontent.com/17327289/230095901-f711dc32-a76a-42c7-a127-a88b09063903.png)